### DISCUSSION TITLE
Abbildungsunterschrift ändern, Abbildungen einfügen

### DISCUSSION THREAD

**Discussion by AKraem** [tudace/tuda_latex_templates#417] [General LaTeX Questions]
Hallo,

leider gibt es die Sprechstunde nicht mehr, dabei habe ich ein paar Fragen, die einfacher zu klären wären. Ich schreib mal ein paar Punkte auf, dich hoffe das ist ok.

1. Meine wichtigste ist, wie kann ich gut Abildungen einbinden. Ich habe zum einen .emf-Dateien aus Matlab und ich habe in Powerpoint Abbildungen erstellt. Wenn ich die als .pdf oder .jpeg abspeicher sehen die im kompilierten pdf nicht gut aus. 
Was wäre da der Tip?

2. Ich würde gerne die Bildunterschriften etwas kleiner machen, wo kann ich das machen?

3. Ich habe in einer Überschrift eine Matheumgebung aufgenommen. Wie bekomme ich die auch in Fett?
\subsection{Probe - $\sigma_{c}=\SI{140}{N}$}}

4. Ich habe eine sehr lange Tabelle, die geht über eine Seite drüber, es wird aber kein automatischer Umbruch gemacht. Geht das?
Du hattest mir da schon mal geholfen wegen der Spaltenbreiten, sodass es jetzt eine 

```
\begin{table}
\renewcommand*{\tabularxcolumn}[1]{m{#1}}
\begin{tabularx}
...
```

geworden ist.

5. Es werden immer wieder Literaturquellen nciht erkannt und dann steht da der BIBtexkey. Ich habe die Stichprobenartig überprüft, die stimmen. 

Vielen Dank
Anna

**Comment by TeXhackse** [comment]
1. Export ist schon richtig. Was heißt „sieht nicht gut aus“? grundsätzlich will man Vektorgrafiken draus machen, also pdf oder den Umweg über eps. 

2. \addtokomafont{caption}{\small}%oder so
3. Am einfachsten für alle Zeichen mit dem bm Paket und dann `\bm` einfügen. 
4. wenn es eine tabularx, die umbrechen können soll werden muss, dann guck dir mal xltabular an. ctan.org/pkg/xltabular
5. Was bedeutet „nicht erkannt“? Läuft Biber richtig durch?

**Reply by AKraem** [reply]
1. Sieht nicht gut aus sollte bedeuten, dass beim reinzoomen die Linien und Schriften in Diagrammen nicht mehr scharf sind. Aus Powerpoint kann ich leider nur als Grafik speichern. Aber wenn ich in diese gespeicherte Grafik reinzoome sieht die besser aus, da ist nichts verschwommen, scharfe Kanten.
5. ich habe über F8 Biber laufen lassen, das wird zwar mit Warnungen beendet, aber es steht Prozess endete normal. Es gibt unerkannte Quellen im Text, die nicht in den Warnungen auftauchen.

**Reply by TeXhackse** [reply]
1. Ah okay. Also ich hatte in einigen Programmen dass der PDF Export nicht von selbst eine Vektorgrafik war. Da kann es sein, dass man rumprobieren muss, welches Format am sinnvollsten ist. Oft ist das so, dass ein Export über eps reicht. 

2. im Log taucht quasi keine Info auf, dass die Quelle nicht existiert? Kannst du mir sonst mal die blg-Datei per Mail schicken? Dann guck ich da mal rein.

**Comment by schoeps** [comment]
Zwei Antworten, die nicht wirklich die Fragen adressieren, aber vielleicht trotzdem hilfreich sind: 

zu 1. meiner Erfahrung nach ist die beste Lösung ein Export der Daten (bei einem normalen Plot z.B. (x,y) Paare in einer csv-Datei) und dann Verwendung von [pgfplots](https://ctan.org/pkg/pgfplots).

zu 3. ich würde aus stilistischen Gründen von Formeln in Überschriften abraten: es sieht komisch aus und ist ggf. auch mißverständlich, denn fast immer hat Fettdruck in Formeln eine inhaltliche Bedeutung, z.B, in vielen Styleguides ist a ein Skalar, aber **a** ein Vektor.

**Comment by TeXhackse** [comment]
@AKraem Meine Mailadresse steht in den Paketen und in der Doku (tuda-ci@peitex.de). Aber ist okay. Ich hab den Kommentar hier, der durch Antwort per Mail erstellt wurde entfernt, damit die Dateien hier nicht rumliegen und antworte per Mail, sobald ich es durchgesehen habe
